src/editor.rs

```diff
@@ -924,4 +925,164 @@ mod tests {
         editor.delete_char();
         assert_eq!(editor.rows[0].chars, "Helo world".as_bytes());
     }
+
+    #[test]
+    fn editor_move_cursor_left() {
+        let mut editor = Editor::default();
+        for b in "Hello world!\nHappy New Year!".as_bytes() {
```

Comment:
Nit: `"Hello world!\nHappy New Year!".as_bytes()` can be simplified as `b"Hello world!\nHappy New Year!"` (and similarly below)

Reply:
Thank you for the feedback. I will commit these changes soon.